test/expected/partials_path_regex.js

```diff
@@ -1,14 +1,15 @@
 this["JST"] = this["JST"] || {};
 
 Handlebars.registerPartial("partial", Handlebars.template({"compiler":[6,">= 2.0.0-beta.1"],"main":function(depth0,helpers,partials,data) {
```

Comment:
> many fixes for testing.

Not fixes. They're temporary workarounds until the next version of Handlebars changes the output slightly and all the `expected` files have to be updated again. Better to just bite the bullet and improve the tests now.